PoshWinRT.sln

```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.20623.1 VSUPREVIEW
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29613.14
```

Comment:
Upgrading the solution to VS2019 was not mandatory, but keeping it declaring VS2013 was erroneous, since we need at least VS2017 to build a csproj at NetSDK format...

So we may roll back to declare VS2017 (15) instead of VS2019 if you want, but I have supposed that in 2020 everybody has already move on VS2019 or VSCode (which works with this new Net SDK format and don't care of this version number in the sln file)